Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -200,6 +200,10 @@ extension PhotonActionSheetProtocol {
             presentShareMenu(url, tab, buttonView, .up)
         }
 
+        let closeTab = PhotonActionSheetItem(title: Strings.AppMenuCloseTabTitleString, iconString: "action_remove") { action in
```

Comment:
I'm of the opinion that this shouldnt be in the page options menu. We've got to be careful about what we add here because we will start scrolling on smaller devices.

Reply:
I've left this as is, as per @tecgirl 's decision.